website/source/docs/drivers/docker.html.md

```diff
@@ -35,8 +35,8 @@ The `docker` driver supports the following configuration in the job specificatio
 ### Port Mapping
 
 Nomad uses port binding to expose services running in containers using the port
-space on the host's interface. For example, Nomad host running on `1.2.3.4` may
-allocate port `22333` to a task, so you would access that service via
+space on the host's interface. For example, a Nomad host running on `1.2.3.4` may
+allocate port `22333` to a service, so you would access that service via
```

Comment:
Hey, I think we want to keep it as a `task`. That has a very specific meaning in Nomad and ports are actually being assigned to tasks.